Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:2
+ADD . /putio-sync
+WORKDIR /putio-sync
+RUN python setup.py install
+CMD ["putiosync"]
```

Comment:
I will probably remove the Dockerfile for now.  I can see its utility to a degree but would rather see it introduced with documentation if it is to be added proper.